Exercise_4/css/partials/primary-navigation/_big-screen.scss

```diff
@@ -0,0 +1,19 @@
+@import '../variables';
+
+/*PRIMARY NAVIGATION*/
+
+@media only screen and (min-width: $big-screen_min-width) {
+  .primary-navigation {
+    display: flex;
+    margin: 0 5rem;
+    width: 30vw;
+  }
+
+  .primary-navigation__item {
```

Comment:
The idea of BEM is to make names shorter through the inheritance. So it should be like this:
```
.primary-navigation {
  ..
  &__item {
    ..
  }
}
```